WordPress/Classes/ViewRelated/Blog/My Site/Header/HomeSiteHeaderViewController+SiteActions.swift

```diff
@@ -108,7 +108,7 @@ private enum MenuItem {
     var title: String {
         switch self {
         case .visitSite: return Strings.visitSite
-        case .shareSite: return Strings.shareSite
+        case .shareSite: return SharedStrings.Button.share + "…"
```

Comment:
Why add... here? I don't think other Share buttons have it.

Reply:
I think I added it because the default `UITextView` also does it for URLs. I'm not sure what's the reasoning though. I removed it in https://github.com/wordpress-mobile/WordPress-iOS/pull/23975.